src/app/views/tour/utils/contextHelpers.ts

```diff
@@ -1,9 +1,13 @@
 import { ContextualMenuItemType } from '@fluentui/react';
+import { getTourSteps } from './blob-steps/getSteps';
 import { componentToTargetMap } from './mapToComponent';
-import { ADVANCED_TOUR, COMPONENT_INFO } from './steps';
-import { ITourContextMenu } from './types';
+import { COMPONENT_INFO } from './steps';
+import { ITourContextMenu } from './types'
+
+
+export const getTargetStepIndex = (target: string, menuType: string, tourSteps: any, beginner: boolean): number => {
+  const ADVANCED_TOUR = getTourSteps(tourSteps, beginner)
```

Comment:
*Nitpick*: The naming convention according to how the rest of the code works depicts a constant we can use in a global scope. e.g actions / URLS.. 

It could then read 
```suggestion
  const advancedTour = getTourSteps(tourSteps, beginner);
```
```suggestion
  const ADVANCED_TOUR = getTourSteps(tourSteps, beginner)
```